htdocs/reception/class/reception.class.php

```diff
@@ -756,7 +756,7 @@ public function getStatusDispatch()
 				// qty wished in order supplier (origin)
 				foreach ($this->commandeFournisseur->lines as $origin_line) {
 					// exclude lines not qualified for reception
-					if (empty($conf->global->STOCK_SUPPORTS_SERVICES) && $origin_line->product_type > 0) {
+					if ((empty($conf->global->STOCK_SUPPORTS_SERVICES) && $origin_line->product_type == 1) || $origin_line->product_type > 1) {
```

Comment:
Can yo tell me if 
`if ((empty($conf->global->STOCK_SUPPORTS_SERVICES) && $origin_line->product_type > 0) || $origin_line->product_type > 1) {`
works for you ?

Reply:
Yes, it work, i've done the change